gammapy/catalog/fermi.py

```diff
@@ -352,13 +357,21 @@ def flux_points(self):
         eflux_ul = compute_flux_points_ul(table['eflux'], table['eflux_errp'])
         table['eflux_ul'][is_ul] = eflux_ul[is_ul]
 
+        # Square root of test statistic
+        ts = self._get_ts_values()
```

Comment:
I think you can just do this and remove the method here:
```
table['sqrt_ts'] = [self.data['Sqrt_TS' + _] for _ in self._ebounds_suffix]
```
TS doesn't have a unit, so I don't think you need to call `Quantity`.